src/test/java/com/rtb/projectmanagementtool/task/TaskControllerTest.java

```diff
@@ -134,16 +157,47 @@ public void testGetTasksFromDs() {
 
     // Get task entities with TaskController
     TaskController taskController = new TaskController(ds);
-    HashSet<TaskData> getTasks = taskController.getTasks(5, "name", "ascending");
+    ArrayList<TaskData> getTasks = taskController.getTasks(5, "name", "ascending");
+
+    // Assert all entities were retrieved
+    Assert.assertEquals("getTask", tasks, getTasks);
+  }
+
+  @Test
+  public void testAddTasks() {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+
+    // Assert no task entities are found
+    Assert.assertEquals(0, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+
+    // Add task entities with TaskController
+    TaskController taskController = new TaskController(ds);
+    taskController.addTasks(new ArrayList<>(Arrays.asList(task1, task2, task3)));
+
+    // Assert 3 entities were added
+    Assert.assertEquals(3, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+  }
+
+  @Test
+  public void testDeleteTasks() {
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
 
-    // Sort getTasks by taskID
-    ArrayList<TaskData> getTasksArrayList = new ArrayList<>();
-    getTasksArrayList.addAll(getTasks);
-    Collections.sort(getTasksArrayList);
+    // Assert no task entities are found
+    Assert.assertEquals(0, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+
+    // Add task entities to ds
+    ds.put(task1.toEntity());
+    ds.put(task2.toEntity());
+    ds.put(task3.toEntity());
 
-    // Assert all 3 entities were retrieved
-    for (int i = 0; i < 3; i++) {
-      Assert.assertTrue("getTask", getTasksArrayList.get(i).equals(tasks.get(i)));
-    }
+    // Assert 3 entities were added
+    Assert.assertEquals(3, ds.prepare(new Query("Task")).countEntities(withLimit(10)));
+
+    // Delete task entities with TaskController
+    TaskController taskController = new TaskController(ds);
+    taskController.deleteTasks(new ArrayList<>(Arrays.asList(taskID1, taskID2, taskID3)));
```

Comment:
It could be more interesting to see this delete some, but not all records.  It'll more clearly show if the code was misconfigured and deletes all records.